saleor/static/dashboard-next/theme.ts

```diff
@@ -17,7 +17,7 @@ export type IThemeColors = Record<
 > & {
   background: Record<"default" | "paper", string>;
 } & {
-  font: Record<"default" | "gray", string>;
+  font: Record<"default" | "gray" | "buttonText" | "textColor", string>;
```

Comment:
`button` and `text` would be sufficient. Btw more descriptive name for `textColor` would be `textButton`